dist/src/assembly/bin/zeneventserver-functions.sh

```diff
@@ -64,6 +68,32 @@ wait_for_startup() {
     return 0
 }
 
+update_schema() {
+    zengc=$ZENHOME/bin/zenglobalconf
+    local dbname=$(${zengc} -p zep-db)
+    local dbtype=$(${zengc} -p zep-db-type)
+    local host=$(${zengc} -p zep-host)
+    local port=$(${zengc} -p zep-port)
+    local user=$(${zengc} -p zep-user)
+    local userpass=$(${zengc} -p zep-password)
+    /opt/zenoss/bin/zeneventserver-create-db --update_schema_only \
+	--dbhost $host --dbport $port \
+        --dbname $dbname --dbtype $dbtype \
+        --dbuser $user --dbpass "$userpass" || return "$?"
+    return 0
+}
+
+wait_and_update_schema() {
+    local port=$1
+    wait_for_startup $port || return $?
+    update_schema || return $?
+    return 0
+}
+
+update_schema_in_background() {
+    ${ZENHOME}/bin/zeneventserver wait_and_update_schema >${ZENHOME}/log/zenupdateschema.log 2>&1 &
```

Comment:
If logged to a file the output would be lost after the container shuts down. Would it be preferable to add it to the existing zeneventserver log?